### ISSUE TITLE
Wrong type argument

### ISSUE THREAD

**Issue by KazamaRuri** [emacs-dashboard/emacs-dashboard#454]
Hello!
I recently upgraded to the latest Dashboard and found that it wasn't loading on startup. 
I try to annotate all the other statements and only left 
```
(use-package dashboard
  :init
  (add-hook 'after-init-hook 'dashboard-open))
```
but return  `apply: Wrong type argument: char-or-string-p, nil` when start emacs.
When i open the dashboard by `M-x dashboard-open`, it also returns `apply: Wrong type argument: char-or-string-p, nil`
i'm using emacs 28.2-2

**Comment by jcs090218** [comment]
Can you post the backtrace here? Thanks!

**Comment by KazamaRuri** [comment]
`debug-on-entry` to dashboad-open
```
Debugger entered--entering a function:
* dashboard-open()
  funcall-interactively(dashboard-open)
  call-interactively(dashboard-open record nil)
  command-execute(dashboard-open record)
  counsel-M-x-action("dashboard-open")
  ivy-call()
  ivy-read("M-x " [org-capture-goto-last-stored 0 org-element-paragraph-separate calendar-forward-week otama dashboard-ls bibtex-entry-right-delimiter ivy-explorer org-indent-drawer sgml-mode ivy-default-view-name help-follow-symbol pcomplete-stub japanese-hiragana-two-byte zmq dired-unmark-all-marks znc uniquify-rename-buffer inactive 0 org-table--descriptor-line raw-text-dos lisp-mode-map info--manual-names 0 tree-widget-button-keymap minibuffer-text-before-history compilation-error-list kmacro-pop-ring1 org-switch-to-buffer-other-window nexus vc-annotate text-property--find-end-backward xref--push-markers 0 About 0 org-agenda-regexp-filter-preset browse-url-generic-args org-agenda-menu-show-matcher use-package-handler/:interpreter 0 dired-do-chmod projectile-unserialize org-get-org-file check-declare xref-group counsel--async-filter Hardlink\.\.\. puny-encode-digit ...] :predicate #f(compiled-function (sym) #<bytecode -0x1ef0d9c037364fc3>) :require-match t :history counsel-M-x-history :action counsel-M-x-action :keymap (keymap (67108908 . counsel--info-lookup-symbol) (67108910 . counsel-find-symbol)) :initial-input nil :caller counsel-M-x)
  counsel-M-x()
  funcall-interactively(counsel-M-x)
  call-interactively(counsel-M-x nil nil)
  command-execute(counsel-M-x)
```

**Comment by jcs090218** [comment]
Use `M-x toggle-debug-on-error` will help us find the culprit! Thanks!

**Comment by KazamaRuri** [comment]
```
Debugger entered--Lisp error: (wrong-type-argument char-or-string-p nil)
  insert(nil " " #("I showed you my source code, pls respond" 0 40 (face dashboard-footer)) "\n")
  apply(insert (nil " " #("I showed you my source code, pls respond" 0 40 (face dashboard-footer)) "\n"))
  dashboard-insert-center(nil " " #("I showed you my source code, pls respond" 0 40 (face dashboard-footer)) "\n")
  dashboard-insert-footer()
  dashboard-insert-startupify-lists()
  #f(compiled-function (&rest _) "Open (or refresh) the *dashboard* buffer." (interactive nil) #<bytecode -0xf58a5dfc954a968>)()
  apply(#f(compiled-function (&rest _) "Open (or refresh) the *dashboard* buffer." (interactive nil) #<bytecode -0xf58a5dfc954a968>) nil)
  dashboard-open()
  funcall-interactively(dashboard-open)
  call-interactively(dashboard-open record nil)
  command-execute(dashboard-open record)
  execute-extended-command(nil "dashboard-open" "dashboard-open")
  funcall-interactively(execute-extended-command nil "dashboard-open" "dashboard-open")
  call-interactively(execute-extended-command nil nil)
  command-execute(execute-extended-command)
  recursive-edit()
  debug(debug)
  debug--implement-debug-on-entry()
  apply(debug--implement-debug-on-entry nil)
* dashboard-open()
  funcall-interactively(dashboard-open)
  call-interactively(dashboard-open record nil)
  command-execute(dashboard-open record)
  execute-extended-command(nil "dashboard-open" "dashboard-open")
  funcall-interactively(execute-extended-command nil "dashboard-open" "dashboard-open")
  call-interactively(execute-extended-command nil nil)
  command-execute(execute-extended-command)
```

**Comment by jcs090218** [comment]
Thanks! Fixed in 770945b2d62014ab949e7472860acb739393d15e.

**Comment by KazamaRuri** [comment]
thank you for your patience
